src/loader/AMASSLoader.py

```diff
@@ -0,0 +1,320 @@
+            "cuda" if torch.cuda.is_available() else "cpu")
+        body_model = BodyModel(bm_path=bm_path, num_betas=num_betas, num_dmpls=num_dmpls, path_dmpl=dmpl_path).to(
+            comp_device)
+        faces = body_model.f.detach().cpu().numpy()
+        return body_model, faces
+
+    @staticmethod
+    def _get_supported_mocap_datasets(taxonomy_file_path, data_path):
+        """
+        get latest updated list from taxonomoy json file about the supported mocap datasets supported in the loader
+        module and update.supported_mocap_datasets list :param taxonomy_file_path: path to taxomomy.json file which
+        contains the supported datasets and their respective paths :param data_path: path to the AMASS dataset root
+        folder :return: None.
+        """
+        if os.path.exists(taxonomy_file_path):
+            with open(taxonomy_file_path, "r") as f:
+                loaded_data = json.load(f)
+                for block in loaded_data:
+                    if "sub_data_id" in block:
+                        sub_dataset_id = block["sub_data_id"]
+                        AMASSLoader.supported_mocap_datasets[sub_dataset_id] = os.path.join(
+                            data_path, block["path"])
+        else:
+            raise Exception(
+                "The taxonomy file could not be found: {}".format(taxonomy_file_path))
+
+    @staticmethod
+    def _clean_up_temp_dir(data_path):
+        """ Cleans up temporary generated poses directory """
+        temp_dir_path = os.path.join(data_path, "generated_poses")
+        if os.path.exists(temp_dir_path):
+            shutil.rmtree(temp_dir_path)
+
+    @staticmethod
+    def _write_body_mesh_to_obj_file(body_represenstation, faces, data_path):
+        """
+        write the generated pose as obj file on the desk.
+        :param body_represenstation: parameters generated from the BodyModel model which represent the obj pose and shape.
+        :param faces: face parametric model which is used to generate the face mesh.
+        :param data_path: path to the AMASS dataset root folder
+        :return: path to generated obj file.
+        """
+        mesh_output_path = os.path.join(data_path, "generated_poses")
+        if not os.path.exists(mesh_output_path):
+            os.mkdir(mesh_output_path)
+        starttime = datetime.now().replace(microsecond=0)
+        # Generate temp objecet with name = timestamp
+        obj_file_name = datetime.strftime(starttime, '%Y%m%d_%H%M') + ".obj"
+        # Write to an .obj file
+        outmesh_path = os.path.join(mesh_output_path, obj_file_name)
+        with open(outmesh_path, 'w') as fp:
+            # convert from tensor to numpy
+            for v in body_represenstation.v[0].detach().cpu().numpy():
+                fp.write('v %f %f %f\n' % (v[0], v[1], v[2]))
+            for f in faces + 1:  # Faces are 1-based, not 0-based in obj files
+                fp.write('f %d %d %d\n' % (f[0], f[1], f[2]))
```

Comment:
This seems quite slow: 

It is probably better to use internal python functions here:
```python
fp.write("\n".join(['f {} {} {}\n'.format(f[0], f[1], f[2]) for f in faces + 1])
```

Same for the vertices.